src/components/Popup.js

```diff
@@ -1,11 +1,20 @@
 export default class Popup {
   constructor({ popupSelector }) {
     this._popupElement = document.querySelector(popupSelector);
+    this._submitButton = this._popupElement.querySelector(
+      ".modal__save-button"
+    );
+    this._popupSelector = popupSelector;
```

Comment:
Needs correction: there is no need to save the popup selector in the class instance